.browserslistrc

```diff
@@ -0,0 +1 @@
+defaults
```

Comment:
1. What is this file about?
2. What does `defaults` refer to?
3. What is the command to get the result of `defaults`?

Reply:
1. A tool to target specific browser version supported in front end. The configuration controls the compiled JavaScript so that the code will be compatible with the selected browser versions.
2. A list of default browser versions listed by browserslist
3. `npx browserslist defaults`